lootbox/api.py

```diff
@@ -111,177 +135,314 @@ async def get_drop_handler(
     )
 
 
+@app.get("/drops/contracts", response_model=List[data.DropperContractResponse])
+async def get_dropper_contracts_handler(
+    blockchain: Optional[str] = Query(None),
+    db_session: Session = Depends(db.yield_db_session),
+) -> List[data.DropperContractResponse]:
+    """
+    Get list of drops for a given dropper contract.
+    """
+
+    try:
+        results = actions.list_dropper_contracts(
+            db_session=db_session, blockchain=blockchain
+        )
+    except NoResultFound:
+        raise DropperHTTPException(status_code=404, detail="No drops found.")
+    except Exception as e:
+        logger.error(f"Can't get list of dropper contracts end with error: {e}")
+        raise DropperHTTPException(status_code=500, detail="Can't get contracts")
+
+    response = [
+        data.DropperContractResponse(
+            id=result.id,
+            blockchain=result.blockchain,
+            address=result.address,
+            title=result.title,
+            description=result.description,
+            image_uri=result.image_uri,
+        )
+        for result in results
+    ]
+
+    return response
+
+
+@app.get("/drops/blockchains")
+async def get_drops_blockchains_handler(
+    db_session: Session = Depends(db.yield_db_session),
+) -> List[data.DropperBlockchainResponse]:
+    """
+    Get list of blockchains.
+    """
+
+    try:
+        results = actions.list_drops_blockchains(db_session=db_session)
+    except NoResultFound:
+        raise DropperHTTPException(status_code=404, detail="No drops found.")
+    except Exception as e:
+        logger.error(f"Can't get list of drops end with error: {e}")
+        raise DropperHTTPException(status_code=500, detail="Can't get drops")
+
+    response = [
+        data.DropperBlockchainResponse(
```

Comment:
I think it's better to just send `List[str]`.

Or better, return type is `data.DropperBlockchainResponse` and `data.DropperBlockchainResponse` has a single `blockchains` field which is `List[str]`.

Reply:
On the other hand, this current schema allows us to enrich each blockchain with chain IDs, RPC endpoints, explorer URLs, etc.

So we should keep the current format.

Reply:
We should add pagination!